docs/index.asciidoc

```diff
@@ -379,7 +405,17 @@ same entry will be updated but entries that existed before but not in the new di
 will remain after the merge; `replace` causes the whole dictionary to be replaced
 with a new one (deleting all entries of the old one on update).
 
+[id="plugins-{type}s-{plugin}-target"]
+===== `target`
+
+  * Value type is <<string,string>>
+  * Default value depends on whether <<plugins-{type}s-{plugin}-ecs_compatibility>> is enabled:
+    ** ECS Compatibility disabled: `"translation"`
+    ** ECS Compatibility enabled: defaults to the same value as `field`
 
+The target field you wish to populate with the translated code.
+Set this to the same value as source `field` causes the plugin to do a substitution,
+in this case filter will always succeed. This will clobber the old value of the source field!
```

Comment:
```suggestion
If you set this value to the same value as source `field`, the plugin does a substitution, and
the filter will succeed. This will clobber the old value of the source field!
```

Reply:
Is that sometimes/always/never preferred behavior? 

If this might be the behavior somebody wants, maybe something like: "The filter will succeed, and the old value of the source field will be replaced. "

Reply:
Thanks, should be fine as you suggested ... mentioning "always succeeds" was unnecessary confusing to start with